packages/sample-controllers/src/sample-gas-prices-service/sample-gas-prices-service-method-action-types.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This file is auto generated by `scripts/generate-method-action-types.ts`.
```

Comment:
One thing I noticed is that method action types for services are not generated automatically. I had to add this one myself. I will make a ticket to fix the script.